stacker/commands/stacker/base.py

```diff
@@ -116,10 +116,10 @@ def get_context_kwargs(self, options, **kwargs):
 
     def add_arguments(self, parser):
         # global arguments that should be available on all stacker subcommands
-        parser.add_argument("-p", "--parameter", dest="parameters",
-                            metavar="PARAMETER=VALUE", type=key_value_arg,
+        parser.add_argument("-var", "--variable", dest="variables",
```

Comment:
The unix nerd in me is freaked out by single dash long options.  Lets just have `--var` :)